modules/serverless-kafka-sasl.adoc

```diff
@@ -27,15 +27,21 @@ Red Hat recommends to enable TLS in addition to SASL.
 [source,terminal]
 ----
 $ oc create secret --namespace <namespace> generic <kafka_auth_secret> \
+  --from-literal=protocol="SASL_SASL" \
   --from-file=ca.crt=caroot.pem \
   --from-literal=password="SecretPassword" \
-  --from-literal=saslType="SCRAM-SHA-512" \
+  --from-literal=sasl.mechanism="SCRAM-SHA-512" \
   --from-literal=user="my-sasl-user"
 ----
 +
 [IMPORTANT]
 ====
-Use the key names `ca.crt`, `password`, and `saslType`. Do not change them.
+Use the key names `protocol`, `ca.crt`, `password`, and `sasl.mechanism`. Do not change them.
```

Comment:
Will this be a breaking change, i.e. if a user has already completed this procedure with the old key names, do they need to rename these as part of the upgrade?